CHANGELOG.MD

```diff
@@ -49,6 +50,6 @@ NOTE: For backwards compatibility, all items under the Added, Changed, and Fixed
 
 - Upgrade to stactools 0.2.1.a2 (supporting PySTAC 1.0.0)
 
-[Unreleased]: <https://github.com/stactools-packages/landsat/compare/v0.2.1..main>
+[Unreleased]: <https://github.com/stactools-packages/landsat/compare/v0.2.2..main>
```

Comment:
This need to be reverted to v0.2.1 as well.

Reply:
Thanks. Fixed.